doc/install/installation.md

```diff
@@ -0,0 +1,346 @@
+
+    sudo -u huginn -H editor Procfile
+
+**Debian only** Install upstart and reboot the system (skip this step on Ubuntu):
+
+    sudo apt-get install -y --force-yes upstart
+    sudo reboot
+    # After you you logged back in go to Huginn installation folder
+    cd /home/huginn/huginn
+
+Export the init scripts using foreman:
+
+    sudo foreman export upstart -a huginn /etc/init
+
+**Note:** You have to re-export the init script every time you change the configuration in `.env`!
+
+### Setup Logrotate
+
+    sudo cp lib/support/logrotate/huginn /etc/logrotate.d/huginn
+
+### Compile Assets
+
+    sudo -u huginn -H bundle exec rake assets:precompile RAILS_ENV=production
+
+### Start Your Huginn Instance
+
+    sudo start huginn
+
+## 7. Nginx
+
+**Note:** Nginx is the officially supported web server for Huginn. If you cannot or do not want to use Nginx as your web server, the wiki has a page on how to configure [apache](https://github.com/cantino/huginn/wiki/Apache-Huginn-configuration).
+
+### Installation
+
+    sudo apt-get install -y nginx
+
+### Site Configuration
+
+Copy the example site config:
+
+    sudo cp lib/support/nginx/huginn /etc/nginx/sites-available/huginn
+    sudo ln -s /etc/nginx/sites-available/huginn /etc/nginx/sites-enabled/huginn
+
+Make sure to edit the config file to match your setup, if you are running multiple nginx sites remove the `default_server` argument from the `listen` directives:
+
+    # Change YOUR_SERVER_FQDN to the fully-qualified
+    # domain name of your host serving Huginn.
+    sudo editor /etc/nginx/sites-available/huginn
+
+Remove the default nginx site, **if huginn is the only enabled nginx site**:
+
+    sudo rm /etc/nginx/sites-enabled/default
+
+**Note:** If you want to use HTTPS, replace the `huginn` Nginx config with `huginn-ssl`. See [Using HTTPS](#using-https) for HTTPS configuration details.
+
+### Test Configuration
+
+Validate your `huginn` or `huginn-ssl` Nginx config file with the following command:
+
+    sudo nginx -t
+
+You should receive `syntax is okay` and `test is successful` messages. If you receive errors check your `huginn` or `huginn-ssl` Nginx config file for typos, etc. as indicated in the error message given.
+
+### Restart
+
+    sudo service nginx restart
+
+# Done!
+
+### Initial Login
+
+Visit YOUR_SERVER in your web browser for your first Huginn login. The setup has created a default admin account for you. You can use it to log in:
+
+    admin
+    password
+
+
+**Enjoy!**
+
+You can use `sudo start huginn` and `sudo stop huginn` to start and stop Huginn.
+
+## Advanced Setup Tips
+
+### Using HTTPS
+
+To use Huginn with HTTPS:
+
+1. In `.env`:
+    1. Set the `FORCE_SSL` option to `true`.
+1. Use the `huginn-ssl` Nginx example config instead of the `huginn` config:
+    1. `sudo cp lib/support/nginx/huginn-ssl /etc/nginx/sites-available/huginn`
+    1. Update `YOUR_SERVER_FQDN`.
+    1. Update `ssl_certificate` and `ssl_certificate_key`.
```

Comment:
Maybe we should show how to setup self-signed certs and make that the recommended path?